cmd/nerdctl/image/image_convert.go

```diff
@@ -89,6 +89,12 @@ func convertCommand() *cobra.Command {
 	cmd.Flags().String("overlaybd-dbstr", "", "Database config string for overlaybd")
 	// #endregion
 
+	// #region soci flags
+	cmd.Flags().Bool("soci", false, "Convert image to SOCI Index V2 format.")
+	cmd.Flags().Int64("soci-min-layer-size", -1, "The minimum size of layers that will be converted to SOCI Index V2 format")
```

Comment:
Should we set the default explicitly instead of passing -1? If we simply want to use SOCI defined defaults, can we simply not pass these options if not set?